homeassistant/loader.py

```diff
@@ -276,7 +275,7 @@ async def async_get_integration_descriptions(
         if integration.integration_type in ("entity", "system"):
             continue
 
-        for integration_type in ("device", "hardware", "helper", "hub", "service"):
+        for integration_type in ("integration", "hardware", "helper"):
```

Comment:
Why was this reverted?

Reply:
Oh nevermind, that shouldn't have been split out and is a fix for a current issue.

Reply:
Because this is a "different" list that was optimized for frontend consumption.